pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -627,6 +627,50 @@ class AnalyticsImpl implements Analytics {
   }
 }
 
+class FakeAnalytics extends AnalyticsImpl {
```

Comment:
I think this should have a dartdoc explaining exactly what is different between this and `AnalyticsImpl`.